backend/src/main/java/de/neuefische/ffmjava221/teamprojekt/backend/employee/EmployeeDTO.java

```diff
@@ -1,7 +1,15 @@
 package de.neuefische.ffmjava221.teamprojekt.backend.employee;
 
-public record UpdateEmployee(
+import org.springframework.data.annotation.Id;
 
+import javax.validation.constraints.NotBlank;
+
+public record EmployeeDTO(
```

Comment:
The name EmployeeDTO could be missleading, since it is very generic... 🤷‍♂️

Reply:
ok ...